prereise/gather/demanddata/transportation_electrification/immediate.py

```diff
@@ -0,0 +1,298 @@
+from prereise.gather.demanddata.transportation_electrification import const
+from prereise.gather.demanddata.transportation_electrification import mileage
+from prereise.gather.demanddata.transportation_electrification import charging
+import numpy as np
+
+
+def immediate_charging(
```

Comment:
This function could use a parameter that specifies the location of the data file, and passes this through to `load_data`, since otherwise it can only be run within a directory that contains the data file with the exact name.